parsing/pcoq.mli

```diff
@@ -29,7 +29,7 @@ module Entry : sig
   val create : string -> 'a t
   val parse : 'a t -> Parsable.t -> 'a
   val print : Format.formatter -> 'a t -> unit
-  val of_parser : string -> (Tok.t Stream.t -> 'a) -> 'a t
+  val of_parser : string -> ((int -> Loc.t) -> Tok.t Stream.t -> 'a) -> 'a t
```

Comment:
Is the `Plexing.location_function` type abbreviation no longer available at this point?

Reply:
It is reachable, only that the right module name is `Gramlib.Plexing`

Reply:
Aha! That was the trick. Will fix.

Reply:
Done.